src/lib.rs

```diff
@@ -62,7 +61,7 @@ fn encode(current: f64, previous: f64, factor: i32) -> Result<String, String> {
     }
     let from_char = char::from_u32((coordinate + 63) as u32).ok_or("Couldn't convert character")?;
     output.push(from_char);
```

Comment:
And we now just push directly onto the pass in string. Saving an allocation and a concatenation.